### DISCUSSION TITLE
加入苦力怕论坛的随机图片API

### DISCUSSION THREAD

**Discussion by githubusera2034324** [Meloong-Git/PCL#3687] [讨论]
`https://klpbbs.com/plugin.php?id=comiis_app_portal`

**Comment by allMagicNB** [comment]
没啥用

**Comment by hejiehao** [comment]
有点像 https://github.com/Hex-Dragon/PCL2/issues/2937

**Comment by wuliaodexiaoluo** [comment]
龙猫应该不会考虑

**Comment by MoYuan-CN** [comment]
访问后会得到
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/379373ff-9b39-473d-8a2c-2f3c316053cd)

**Reply by githubusera2034324** [reply]
这只是上面网址栏的网址，不是真正的API

**Comment by Xiaobei08** [comment]
API应该是这个:https://api.klpbbs.com/api/img/?type=%E4%BA%8C%E6%AC%A1%E5%85%83&back=img-------- 原始邮件 --------发件人： 凌云 ***@***.***>日期： 2024年4月14日周日 20:07收件人： Hex-Dragon/PCL2 ***@***.***>抄送： Subscribed ***@***.***>主    题： Re: [Hex-Dragon/PCL2] 加入苦力怕论坛的随机图片API (Discussion #3687)
访问后会得到
image.png (view on web)

**Reply by wuliaodexiaoluo** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/160083863/1cc0c1b2-4cda-4be0-8fa5-bc32e575468d)
这请求过去是空返回啊

**Reply by MoYuan-CN** [reply]
楼主描述的应当是 [https://api.klpbbs.com/api/img/?type=%E4%BA%8C%E6%AC%A1%E5%85%83&back=img](https://api.klpbbs.com/api/img/?type=%E4%BA%8C%E6%AC%A1%E5%85%83&back=img) 
邮件回复头 `-------- 原始邮件 --------` 左边那部分被算作链接了

**Reply by MoYuan-CN** [reply]
不过无论如何，类似 #2937 的，PCL 不会支持以 url 加载背景图片

**Reply by wuliaodexiaoluo** [reply]
不是不会支持，#2397 这个是因为没有 API 可以调用+不影响使用
#215 #216
还没开始做
不过 `二次元图片` API 是绝对不可能加的
https://github.com/MFn233/PCL-Mainpage/issues/2
你不知道也不可能知道这个 API 里面到底有没有 R18 图 ~保不准就刷出来了~

**Comment by githubusera2034324** [comment]
破案了
`https://api.klpbbs.com/api/img/?type=%E4%BA%8C%E6%AC%A1%E5%85%83&back=img
